.github/workflows/ci.yml

```diff
@@ -117,7 +117,7 @@ jobs:
     container:
       image: "kiwix/kiwix-build_ci:${{matrix.image_variant}}-31"
     steps:
-    - name: Checkout code
```

Comment:
Why this name change ?

We are in the CI, so we do nothing manually.

Reply:
This was because we're checking out via script instead of using the github actions/checkout. But mostly I just needed to make another commit happen to get the CI to cycle for my testing...I can revert if you prefer.

Reply:
> I can revert if you prefer.

Yes please. The fact we use our own script or `actions/checkout` is a implementation details.
The "Manually" in the name implies (for me) that we need some kind of manual intervention.